### DISCUSSION TITLE
Widget Layer

### DISCUSSION THREAD

**Discussion by RRozi** [flet-dev/flet#3902] [Q&A]
### Question

And I have this design
A page with widgets and on top of them another Container with visible = False
One of the widgets on the page is an input field, and when you enter into this field, that same Conatiner becomes visble = True, and overlaps all elements on the page, so I need that same input field not to overlap at this moment, I tried to use all possible options I looked all over the internet and couldn't find anything like it
p.s: Adding an input field to the overlay will not help me, since the input field itself is located with other widgets in the ListView

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ap4499** [comment]
Could you please provide a code example? (most minimal possible code to reproduce what you are seeing)
